pom.xml

```diff
@@ -81,11 +58,6 @@
 			<version>0.10.5</version>
 			<scope>runtime</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.postgresql</groupId>
-			<artifactId>postgresql</artifactId>
-			<scope>runtime</scope>
-		</dependency>
```

Comment:
I left only the jwt and spring security dependencies so that the repo is independent of databases and tests, so that the person who wants to add a login with jwt simply reuses this code and dependencies in their project